e2e/storybook-angular/src/storybook-angular.test.ts

```diff
@@ -2,321 +2,120 @@ import {
   checkFilesExist,
   cleanupProject,
   getPackageManagerCommand,
+  getSelectedPackageManager,
   isNotWindows,
   killPorts,
-  newProject,
-  readFile,
   runCLI,
   runCommand,
+  runCreateWorkspace,
   runCypressTests,
   tmpProjPath,
   uniq,
-  updateJson,
 } from '@nrwl/e2e/utils';
 import { writeFileSync } from 'fs';
 
 describe('Storybook for Angular', () => {
-  let proj: string;
+  const previousPM = process.env.SELECTED_PM;
+  const packageManager = getSelectedPackageManager() || 'yarn';
+  const proj = uniq('proj');
+  const appName = uniq('app');
 
   beforeAll(() => {
-    proj = newProject();
-
-    // TODO(jack): Overriding enhanced-resolve to 5.10.0 now until the package is fixed.
-    // See: https://github.com/webpack/enhanced-resolve/issues/362
-    updateJson('package.json', (json) => {
-      if (process.env.SELECTED_PM === 'yarn') {
-        json['resolutions'] = {
-          'enhanced-resolve': '5.10.0',
-        };
-      } else if (process.env.SELECTED_PM === 'npm') {
-        json['overrides'] = {
-          'enhanced-resolve': '5.10.0',
-        };
-      } else {
-        json['pnpm'] = {
-          overrides: {
-            'enhanced-resolve': '5.10.0',
-          },
-        };
-      }
-      return json;
+    process.env.SELECTED_PM = 'yarn';
+    runCreateWorkspace(proj, {
+      preset: 'angular-monorepo',
+      appName,
+      style: 'css',
+      packageManager,
     });
-    runCommand(getPackageManagerCommand().install);
-  });
-
-  afterAll(() => cleanupProject());
 
-  it('should not overwrite global storybook config files', () => {
-    const angularStorybookLib = uniq('test-ui-lib-angular');
     runCLI(
-      `generate @nrwl/angular:lib ${angularStorybookLib} --no-interactive`
-    );
-    runCLI(
-      `generate @nrwl/angular:storybook-configuration ${angularStorybookLib} --generateStories --no-interactive`
+      `generate @nrwl/angular:storybook-configuration ${appName} --generateStories --storybook7Configuration --no-interactive`
     );
 
-    checkFilesExist(`.storybook/main.js`);
-    writeFileSync(
-      tmpProjPath(`.storybook/main.js`),
-      `
-        module.exports = {
-          stories: [],
-          addons: ['@storybook/addon-essentials'],
-        };
-
-        console.log('hi there');
-      `
-    );
+    runCommand(getPackageManagerCommand().install);
```

Comment:
Install should be part of the `@nrwl/angular:storybook-configuration`. If it's not we need to fix it there

Reply:
Oops yes that was left there from the `enhance` package addition!